src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -1479,6 +1619,57 @@ .maxstack  1
             AssertEx.Equal(expectedMembers, actualMembers);
         }
 
+        [Fact]
+        public void EmptyRecord_01_RecordClass()
+        {
+            var src = @"
+record class C();
```

Comment:
:memo: The `class` keyword is ignored by the compiler. So I just picked a couple of tests and duplicated them to give `record class` some coverage.